core/src/main/java/de/danoeh/antennapod/core/util/playback/ExternalMedia.java

```diff
@@ -99,6 +108,19 @@ public void loadMetadata() throws PlayableException {
             e.printStackTrace();
             throw new PlayableException("NumberFormatException when reading duration of media file");
         }
+
+        String dateStr = mmr.extractMetadata(MediaMetadataRetriever.METADATA_KEY_DATE);
```

Comment:
How about unifying this with the code in `LocalFeedUpdater`? Not sure if there is a good place to put this, so if you don't have a good idea it can also be merged as-is.

Reply:
not sure if I have a great idea.